spec/spec_helper.rb

```diff
@@ -0,0 +1,42 @@
+$LOAD_PATH.unshift File.expand_path('../../lib', __FILE__)
+require 'log_decorator'
+require "log4r"
+
+module TestLog
+  class ConsoleFormatter < Log4r::Formatter
+    def format(event)
+      (event.data.is_a?(String) ? event.data : event.data.inspect) + "\n"
+    end
+  end
+
+  def self.log_init(log_level = 'DEBUG')
+    log = Log4r::Logger.new 'toplog'
+    log.level = case log_level
+                when 'ERROR' then Log4r::ERROR
+                when 'WARN'  then Log4r::WARN
+                when 'INFO'  then Log4r::INFO
+                when 'DEBUG' then Log4r::DEBUG
+                else              Log4r::OFF
+                end
+    Log4r::IOOutputter.new('err_console', sio, :formatter => ConsoleFormatter)
+    log.add 'err_console'
+    LogDecorator.logger = log
+  end
+
+  def self.sio
+    @sio ||= StringIO.new
+  end
+end
+
+TestLog.log_init
+
+class TestClass1
```

Comment:
Oh weird... probably because an anonymous class doesn't have a name, and the prefixer thing probably didn't like it.  Yeah, we can research this later as it's all under the covers.